operations/deploy/setup_common.py

```diff
@@ -28,6 +28,25 @@
         )
     )
     else []
+    + (
+        ["dffml_feature_git"]
+        if not any(
+            list(
+                map(
+                    os.path.isfile,
+                    list(
+                        map(
+                            lambda syspath: os.path.join(
+                                syspath, "dffml_feature_git"
```

Comment:
```suggestion
                                syspath, "dffml_feature_git.egg-link"
```

Reply:
See: https://intel.github.io/dffml/master/contributing/codebase.html#checking-for-development-version

Reply:
Thanks,i was gonna ask what this is in the next meeting!!